README.md

```diff
@@ -1,548 +1,309 @@
+
+## EVM example
+```typescript
+import { ethers } from 'ethers'
+import {
+  ERC20__factory,
+  Bridge__factory,
+} from "@buildwithsygma/sygma-contracts"
+import { erc20Transfer } from '@buildwithsygma/sygma-sdk-core/EVM'
+// Prepare instances of evm contracts and ethers provider
+const provider = new ethers.providers.Web3Provider(window.ethereum)
+const tokenInstance = ERC20__factory.connect('0x1234567890123456789012345678901234567890', provider)
+const bridgeInstance = Bridge__factory.connect('0x1234567890123456789012345678901234567890', provider)
+// Prepare params
+const params = {
+  amountOrId: '100',
+  recipientAddress: '0x1234567890123456789012345678901234567890',
+  tokenInstance, // ERC20 instance
+  bridgeInstance, // Bridge instance from the sygma-contracts
+  handlerAddress: '0x0987654321098765432109876543210987654321',
+  domainId: '1',
+  resourceId: '0x000000000000000001',
+  feeData: { ... }, // fee data
+  provider,
+  overrides: { gasLimit: 1000000 } // optional
+}
+const transaction = await erc20Transfer(params)
+// wait for the transaction to be mined
+const receipt = await transaction.wait(3)
+// get the deposit event
+const depositEvent = getDepositEvent(receipt)
+// Log the deposit nonce (unique identifier for the transfer)
+console.log("depositNonce", depositEvent.args.depositNonce)
+```
 
+## Substrate example
+```typescript
+import { web3FromAddress } from "@polkadot/extension-dapp"
+import { deposit } from '@buildwithsygma/sygma-sdk-core/Substrate'
+// Prepare signer for the transaction
+const injector = await web3FromAddress(currentAccount.address);
+// const api;  ApiPromise instance
+// const asset; MiltiAsset instance
+console.log(asset)
+// output:
+// {
+//   concrete: {
+//     parents: 1,
+//     interior: {
+//       x3: [
+//         { parachain: 2004 },
+//         { generalKey: "0x7379676d61" }, // sygma
+//         { generalKey: "0x75736463" }, // usdc
+//       ],
+//     },
+//   },
+// }
+const unsub = await deposit(api, asset, amount, domainId, address)
+  .signAndSend(currentAccount.address, { signer: injector.signer }, result => {
+     handleTxExtrinsicResult(api, result, unsub, {
+      onDepositEvent: (depositData) => {
+        // Log the deposit nonce (unique identifier for the transfer)
+        console.log("depositNonce", depositData.depositNonce)
+      },
+     });
+   });
+```
+## React app examples
+
+We have prepared a few examples of how to use the SDK in a React app. You can find them in the [examples](./examples) folder.
+
+- ✅ [evm-to-substrate-react-example](./examples/evm-to-substrate-react-example) - Example of bridging ERC20 tokens from EVM to Substrate
+- ✅ [substrate-to-evm-react-example](./examples/substrate-to-evm-react-example) - Example of bridging asset from Substrate to EVM
+## Getting started
+### Prerequisites
+- Node.js >= 16.0.0 for server-side usage
+- ethers.js >= 5.0.0 for EVM usage
```

Comment:
Should `ethers.js` be listed as pre-requisite even if this is a dependency we are using internally?

Reply:
Yes, because we assume that some kind of web3 provider is created by something like: 
`const providerOrSigner = new ethers.providers.Web3Provider(window.ethereum)`